assets/sass/styles.scss

```diff
@@ -9,7 +9,7 @@ body {
   background-color: var(--bg-neutral);
   font-size: 1.8rem;
   overflow-x: hidden;
-  margin: 0;
+  margin: 0 0 7rem 0;
```

Comment:
The margin should only be applied if we are in mobile view and on a page where the mobile navigation is visible. For example not on the General Information page or on desktop devices.